src/RichText/RichText.php

```diff
@@ -62,11 +62,20 @@ public static function getSafeHtml(?string $content, bool $encode_output_entitie
 
         $content = self::normalizeHtmlContent($content, true);
 
+        // Extract <img> id attribute
+        preg_match_all('/<img[^>]*id="[^"]*"[^>]*>/', $content, $matches);
+        $img_tags = $matches[0];
+
        // Remove unsafe HTML using htmLawed
         $config = Toolbox::getHtmLawedSafeConfig();
         $config['keep_bad'] = 6; // remove invalid/disallowed tag but keep content intact
         $content = htmLawed($content, $config);
 
+        // Reinsert <img> id attribute
+        foreach ($img_tags as $img_tag) {
+            $content = preg_replace('/<img[^>]*>/', $img_tag, $content, 1);
+        }
+
```

Comment:
Could you explain this change ?

Also, what will happen if there are multiple images ? As far as I understand the code, all images will be replaced by the last found image.

Reply:
Line 72: `$content = htmLawed($content, $config);`

`$content` loses the image ID attribute after the 1st ticket, so I try to restore it. I doubt this is the right approach, but I haven't had any other ideas.